app/src/main/java/org/mozilla/fenix/home/HomeFragment.kt

```diff
@@ -554,10 +557,26 @@ class HomeFragment : Fragment() {
             recommendPrivateBrowsingShortcut()
         }
 
+        // In-app review prompt
+        requireContext().settings().incrementNumTimesOpenedAfterInstall()
```

Comment:
We'll probably want to move this into something that waits till after first paint (We may want to move this to FenixApplication `initVisualCompletenessQueueAndQueueTasks`. @mcomella is this the correct place?)

Reply:
Yes.

Note: You can also add tasks from anywhere in the codebase by accessing the queue through Components but this doesn't seem like a use case for it.